### PR TITLE
Remove hardcoded expiry from content and behaviour; disable the file uploader with an environment variable

### PR DESCRIPTION
**Pull request by faissalMT**
**What**
The notice on the front page will now present the correct expiry date to the user based on what is set in the `EXPIRATION_DATE` environment variable
Screenie_16

**Why**  
To avoid mistakes that may arise as a result of needing to change the expiry date in multiple places.

**Anything else?**
I have added the dateformat library, and added a flag that allows me to disable the file uploader instead of having to manually comment it out and potentially end up committing that

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-HACKNERY_AUTH_URL=
-POSTCODE_LOOKUP_URL=
-POSTCODE_LOOKUP_APIKEY=
-URL_PREFIX=
+#This file outlines the environment variables this application needs
+#note that anything not commented will be loaded by the application
+
+#GOV_NOTIFY_API_KEY=
+#EMAIL_APPLICATION_RECEIVED_TEMPLATE_ID=
+#HACKNERY_AUTH_URL=
```
Comment (yusufsheiqh) [inline .env.sample:6]: This is a typo that we may or may not want to fix at some point.

**Comment by mattbee** [comment]
My only question on this and why it was hardcoded is that because this value comes from CircleCI in the deployment pipeline, was this hardcoded so that staging and production can be different values. Do you think having a `EXPIRATION_DATE_STAGING` and `EXPIRATION_DATE_PRODUCTION` will solve that problem?

File: pages/index.js [hunk 2]
```diff
@@ -47,8 +48,9 @@ export default function Home({ date, expirationDate }) {
       )}
       <WarningText>
         Please note that applications for the Discretionary Business Grant close
-        at 23:59 on Friday 28th August 2020. Any applications submitted after
-        this time will be invalid.
+        at {dateFormat(new Date(expirationDate), 'h:MM:ss TT')} on{' '}
+        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS, yyyy')}. Any
```
Comment (yusufsheiqh) [inline pages/index.js:52]: I think following the GOV.UK guidelines for writing out dates is good practice; https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

```suggestion
        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS yyyy')}. Any
```

**Comment by faissalMT** [comment]
I presume it was hardcoded because of time constraints and some code was missed as a result, I don't see any reason to have multiple environment variables for this, changing it as needed should suffice imo

**Review by yusufsheiqh — COMMENTED** [review]
Everything else looks good to me, but is there a reason to comment out the environment variables? It is a sample file after all.

**Comment by faissalMT** [comment]
I commented it out because it was overriding my local `.env` otherwise

File: .env.sample (hunk 1 above)
Comment (faissalMT) [reply .env.sample:6]: I'd like to do that I'm just not sure how far the typo's propagated

**Comment by mattbee** [comment]
What I mean is that up until Friday we don't want the form to be publicly accessible, even if it isn't working yet, however this way we can't have a different applications open state for staging and prod, or am I missing something? Or do we just not deploy to production until Friday?

**Comment by faissalMT** [comment]
When you deploy to prod you can just set a different value for the environment variable. Is there a reason why you would have to use the same environment variables for both staging and prod?

**Comment by mattbee** [comment]
If that's the case and we can set different env vars for prod on deploy, this looks good to me, I didn't realise we could do that :)

**Review by sirLisko — COMMENTED** [review]
@faissalMT you have committed the `package-lock` but the project is using yarn. I was scared by the +15k lines of changes 😅

File: pages/index.js [hunk 3]
```diff
@@ -47,8 +48,9 @@ export default function Home({ date, expirationDate }) {
       )}
       <WarningText>
         Please note that applications for the Discretionary Business Grant close
-        at 23:59 on Friday 28th August 2020. Any applications submitted after
-        this time will be invalid.
+        at {dateFormat(new Date(expirationDate), 'h:MM:ss TT')} on{' '}
+        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS yyyy')}. Any
```
Comment (sirLisko) [inline pages/index.js:52]: Not a big deal but probably we don't need a 3rd party dependency https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

**Pull request merged**